zstacklib/zstacklib/utils/fileUploader.py

```
@@ -0,0 +1,289 @@
+        self.upload_slice(req.body, upload_param, task)
+
+        if task.allow_image_completing():
+            self.complete_upload(task)
+
+    def get_upload_param(req_header):
+        # type: (dict[str, str]) -> UploadParam
+
+        def get_long_field(key, default=None):
+            v = req_header.get(key, default)
+            try:
+                lv = long(v)
```

Ruff (0.13.1)

`long` may be undefined, or defined from star imports

(F405)